server/src/rest/index.ts

```diff
@@ -1,12 +1,13 @@
 import { Express, Router } from 'express';
 import { messages } from '../core';
 import { errorHandler, error404Handler } from './middleware';
-import { root, users, donations, transactions, invitations, refunds, stats } from './routes';
+import { root, users, donations, transactions, invitations, refunds, stats, anonymous } from './routes';
 
 export function mountRestApi(server: Express, apiRoot: string) {
   const router = Router();
 
   router.use('/users', users);
+  router.use('/anonymous', anonymous);
```

Comment:
I find it strange to create a new route just for anonymous, instead of putting the endpoint under donations.